php_phongo.c

```diff
@@ -1870,7 +1870,23 @@ mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri, zval *dri
 		php_phongo_populate_default_ssl_ctx(ctx, driverOptions);
 	}
 
-	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s, libbson-%s", MONGODB_VERSION_S, MONGODB_STABILITY_S, MONGOC_VERSION_S, BSON_VERSION_S);
+	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
+		MONGODB_VERSION_S,
+		MONGODB_STABILITY_S,
+		MONGOC_VERSION_S,
+#ifdef HAVE_SYSTEM_LIBMONGOC
```

Comment:
No worries! I've missed more than my fair share of Windows errors :)

https://github.com/mongodb-labs/mongo-php-driver-prototype/commit/59177872a87324342d336052008c0fd3ac777747 should fix it.